components/common/StreamField.tsx

```diff
@@ -274,16 +278,30 @@ function StreamFieldBlock(props: StreamFieldBlockProps) {
               xl={getColSize('xl')}
               lg={getColSize('lg')}
               md={getColSize('md')}
+              style={{
+                position: 'relative',
+              }}
             >
-              <img
-                src={block.image?.renditionUncropped?.src}
-                alt={block.image?.altText}
+              <div
                 style={{
-                  display: 'block',
-                  width: '100%',
-                  marginBottom: theme.spaces.s600,
```

Comment:
Should we keep this `marginBottom`  on the new `div` element?

Reply:
We should! thanks @woodwoerk !